scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -178,7 +178,7 @@
               <td>{{ gene.hgnc_id }}</td>
 
               <td>{{ gene.disease_associated_transcripts|join(', ') }}</td>
-              <td>{{ 'Yes' if gene.reduced_penetrance }}</td>
+              <td>{{ gene.reduced_penetrance }}</td>
```

Comment:
I'm not 100% sure about the changes in this PR. I think it's good to include also reduced penetrance = False, because it could be important info when creating the panels from PanelApp. But I'm concerned about an old bug when creating gene panels. The bug affected all genes of a panel and the effect was that reduced penetrance was set to False (instead of the key being missing) whenever the user didn't specify the field for the gene. Introducing this change we might have a situation like this, which is extremely confusing 🤔 

<img width="762" alt="image" src="https://github.com/user-attachments/assets/e42d8c93-7e5c-4cc9-961b-7c1e7ab13543" />